saskatoon/member/forms.py

```diff
@@ -220,3 +223,20 @@ def save(self):
         instance.save()
 
         return instance
+
+
+class PasswordChangeForm(auth_forms.PasswordChangeForm):
+
+    def __init__(self, user, *args, **kwargs):
+        super().__init__(user, *args, **kwargs)
+
+        # replace widgets with placeholder values to fit Notika theme
+        self.fields['old_password'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'current-password', 'placeholder': 'Old password', 'autofocus': True })
+        self.fields['new_password1'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'new-password', 'placeholder': 'New password'})
+        self.fields['new_password2'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'new-password', 'placeholder': 'Confirm password'})
+
+    def save(self, commit=True):
+        instance = super().save(False)
```

Comment:
just curious, what happens if `old_password == new_password1 == new_password2` ? does the form throw an error?

Reply:
no error - good catch! i just left the password checks to `auth.PasswordChangeForm` and `AUTH_PASSWORD_VALIDATORS` in `settings.py`. 

I'll see where I can fit another vaildator

Reply:
added password unchanged validator